styles/Home.module.css

```diff
@@ -4,7 +4,7 @@
   position: relative;
   padding-top: 12vh;
   overflow: auto;
-  background-color: var(--background);
+  background-color: white;
```

Comment:
Please keep the var(--background);
But change it's value in the globals.css file